src/EnergyPlus/ResultsFramework.cc

```diff
@@ -377,7 +377,7 @@ namespace ResultsFramework {
         return variableMap.at(lastVarID);
     }
 
-    void DataFrame::newRow(const int month, const int dayOfMonth, int hourOfDay, int curMin)
+    void DataFrame::newRow(const int month, const int dayOfMonth, int hourOfDay, int curMin, int calendarYear)
```

Comment:
Add a calendar year argument. It will only be used if the ISO 8601 format is requested.

Reply:
OK, so always passed in but just not always used internally.  Got it.